src/tests/DbgShim.UnitTests/ConfigFiles/Windows/Debugger.Tests.Config.txt

```diff
@@ -1,15 +1,14 @@
 <Configuration>
   <Import ConfigFile="Debugger.Tests.Common.txt" />
-  <DotNetRoot Condition="'$(TargetArchitecture)' != 'x86'">$(RepoRootDir)\.dotnet-test</DotNetRoot>
-  <DotNetRoot Condition="'$(TargetArchitecture)' == 'x86'">$(RepoRootDir)\.dotnet-test\x86</DotNetRoot>
-  <Import ConfigFile="$(DotNetRoot)\Debugger.Tests.Versions.txt" />
+  <DotNetRoot Condition="'$(TestDotNetRoot)' != ''">$(TestDotNetRoot)</DotNetRoot>
```

Comment:
The SOS tests have hard-coded $(RepoRootDir)/.dotnet-test as DotNetRoot. Any reason DbgShim tests are doing all this parameterization and the SOS tests are essentially hard-coding it? It seems like we'd want all our tests looking for DotNetRoot in the same place.